recipe_scrapers/__init__.py

```diff
@@ -332,6 +333,9 @@
     AltonBrown.host(): AltonBrown,
     AmazingRibs.host(): AmazingRibs,
     AmbitiousKitchen.host(): AmbitiousKitchen,
+    AmericasTestKitchen.host(): AmericasTestKitchen,
+    AmericasTestKitchen.host(domain="cooksillustrated.com"): AmericasTestKitchen,
+    AmericasTestKitchen.host(domain="cookscountry.com"): AmericasTestKitchen,
```

Comment:
Have you checked a few recipes from these also-ATK domains to check that the code handles them reasonably too?

Reply:
> Have you checked a few recipes from these also-ATK domains to check that the code handles them reasonably too?

From what I can tell all of the domains point to the same page.  I have a version of this scraper as part of Tandoor for about two years and haven't run into any issues.

Reply:
Ok, understood (I think!).  So:

* The domains are all ATK domains.
* Some recipes exist on all three of them.
* The non-ATK.com addresses redirect to the main ATK domain.

In other words: people may input URLs that are either the aliased/redirected-from domains, but in all cases the eventual page is the main `AmericasTestKitchen.host()` (zero arguments) value.

I _think_ that's OK - I'm trying to think of any edge cases around it.. none I can think of so far.

Looking at the code from the previous version in Tandoor: it looks like CooksIllustrated was the original version, and then later the other ATK sites were added.  Should we retain some of that history by renaming the scraper to `CooksIllustrated` here too?

Reply:
That's almost right, the only difference is that all recipes can be reached but typing any of the three domains and the recipe slug but will get redirected to AmericasTestKitchen.host().

I'm fine with renaming it.  I started that way and decided to switch since they seem to be using America's Test Kitchen as their primary brand now.

Reply:
Ok, that all makes sense.  And yep, I think it's a good opportunity to normalize to whatever name they prefer now -- although I tend to like keeping some of the history around too.  I suppose anyone that curious might find this pull request discussion.